Report.md

```diff
@@ -68,12 +68,21 @@ git diff ...
 ### BellmanFord::go
 Plan for refactoring complex code: Our plan to refactor this complex code is to create auxiliary functions for specific sub-tasks.
 
-Estimated impact of refactoring (lower CC, but other drawbacks?) : This refactoring considerably lower the CC of the function by taking it from 13 to 7, which is a 46% decrease. Another advantage is that we can test each sub-function seperately which is easier. However, if an input of a function relies on the output of another, then a change to a previous function could obviously destroy functionality in the second function.
+Estimated impact of refactoring (lower CC, but other drawbacks?) : This refactoring considerably lowers the CC of the function by taking it from 13 to 7, which is a 46% decrease. Another advantage is that we can test each sub-function seperately which is easier. However, if an input of a function relies on the output of another, then a change to a previous function could obviously destroy functionality in the second function.
 
 Carried out refactoring (optional, P+): Refactor of the BellmanFord::go function.
 
 Commit link : https://github.com/einbergisak/Assignment3/pull/33/commits/5e163af002e1abeb940a0d9839907794e3dee356
 
+### BinarySearch2dArray::BinarySearch
+Plan for refactoring complex code: Our plan to refactor this complex code is to create auxiliary functions for specific sub-tasks.
+
+Estimated impact of refactoring (lower CC, but other drawbacks?) : This refactoring considerably lowers the CC of the function by taking it from 11 to 4, which is a 64% decrease. Another advantage is that we can test each sub-function seperately which is easier. However, if an input of a function relies on the output of another, then a change to a previous function could obviously destroy functionality in the second function.
```

Comment:
both here and above, the question is essentially "how much impact is estimated", but the answer is stating a result that has already happened. Maybe you meant to put your addition under "carried out refactoring"?